src/importer-api.js

```diff
@@ -4,5 +4,13 @@ import { post } from 'axios'
 import type { ImporterApi } from 'icarus-backend'; // eslint-disable-line
 
 export default (importerSendTxEndpoint: string): ImporterApi => ({
-  sendTx: tx => post(importerSendTxEndpoint, tx),
+  sendTx: tx => post(
+    `${importerSendTxEndpoint}/api/submit/tx`,
```

Comment:
importerSendTxEndpoint is not an accurate name anymore as this is dedicated tx submit service. So I would name it `submitTxApiHost` or something like that to make it clearer